src/Object/Customer.php

```diff
@@ -94,4 +104,41 @@ public function getMarkupKey()
     {
         return 'nosto_customer';
     }
+
+    /**
+     * @return null|string
+     */
+    public function getCustomerGroup()
+    {
+        return $this->customerGroup;
+    }
+
+    /**
+     * @param string $customerGroup
+     */
+    public function setCustomerGroup($customerGroup)
+    {
+        $this->customerGroup = $customerGroup;
+    }
+
+    /**
+     * @return Subscription
+     */
+    public function getSubscription()
+    {
+        return $this->subscription;
+    }
+
+    /**
+     * @param string $name
+     * @param \DateTime $startDate
+     * @throws \Nosto\NostoException
+     */
+    public function setSubscription($name, $startDate)
```

Comment:
This must only allow one argument and the type must be `Nosto\Object\Subscription` . This would need be a simple setter rather than "builder". We can add helper / builders later on if needed.